tools/eol_cve/index.js

```diff
@@ -0,0 +1,116 @@
+const vulnerabilities = require('../../vuln/core/index.json');
+const { createWriteStream } = require('fs');
+const { format } = require('fast-csv');
+const { resolve } = require('path');
+const semver = require('semver');
+const nv = require('@pkgjs/nv');
+const path = require('path');
```

Comment:
```suggestion
'use strict';

const vulnerabilities = require('../../vuln/core/index.json');
const { createWriteStream } = require('fs');
const { format } = require('fast-csv');
const { resolve } = require('node:path');
const semver = require('semver');
const nv = require('@pkgjs/nv');
```

Path is already being imported, so the duplicate import could be removed?